src/components/presenters/CVoxel/visualizerPresenter.tsx

```diff
@@ -1,21 +1,70 @@
 import * as THREE from "three";
-import { FC, useRef, useState, useMemo } from "react";
+import { FC, useRef, useState, useMemo, useEffect } from "react";
 import { Canvas, useFrame } from "@react-three/fiber";
 import {
   OrbitControls,
   PerspectiveCamera,
   Plane,
   softShadows,
 } from "@react-three/drei";
-import { CVoxelVisType, CCubeType } from "@/interfaces/cVoxelType";
+import {
+  CVoxelVisType,
+  CCubeType,
+  CVoxelThree,
+  CVoxel,
+  CVoxels,
+  initCVoxel,
+} from "@/interfaces/cVoxelType";
 import CVoxelPresenter from "./CVoxelPresenter";
+import useVoxStyler from "@/hooks/useVoxStyler";
+import useStacker from "@/hooks/useStacker";
+import { useCVoxelRecord, useCVoxelsRecord } from "@/hooks/useCVoxel";
+import { faRectangleXmark } from "@fortawesome/free-solid-svg-icons";
+import { useCore } from "@self.id/framework";
+
+type Props = {
+  did?: string[];
+  account?: boolean;
+};
+
+const VisualizerPresenter: FC<Props> = (Props) => {
+  //const CVoxelsRecords = useCVoxelsRecord(Props.did);
+  //const ids = CVoxelsRecords.content?.cVoxels.map((vox) => vox.id);
+  const core = useCore();
+  const [cVoxels, setCVoxels] = useState<CVoxel[]>([]);
+
+  //const voxelVis: CVoxelVisType[] = useVoxStyler(cVoxels);
+  const stackedVoxels: (CVoxelThree | undefined)[] = useVoxStyler(cVoxels);
+
+  const cCollectionRef = useRef<THREE.Group>(new THREE.Group());
+  const offset = new THREE.Vector3(0, 0, 0);
+
+  useEffect(() => {
+    const voxelsTemp: CVoxel[] = [];
+    const func = async () => {
+      if (Props.did != undefined) {
+        console.log("did =", Props.did);
+
+        for (let i = 0; i < Props.did!.length; i++) {
+          const voxel = await core.tileLoader.load<CVoxel>(Props.did[i]);
+          voxelsTemp.push(voxel.content);
+        }
+      }
+      setCVoxels(voxelsTemp);
+      console.log("useEffectnonaka", cVoxels);
```

Comment:
As I commented on the previous PR, I would appreciate it if you could remove unnecessary console.logs, comment-outs, etc.

Reply:
My apologies. I removed unnecessary console.logs and comments.